superdesk/publish/subscriber_token.py

```diff
@@ -4,6 +4,7 @@
 from datetime import timedelta
 from superdesk.utc import utcnow
 from superdesk.utils import get_random_token
+from content_api import MONGO_PREFIX
```

Comment:
yep I wasn't sure how to do this, but imo this way it's kind of explicit that it manages a content api resource. any suggestions how to make this better?